galley/pkg/source/fs/source.go

```diff
@@ -351,14 +357,14 @@ type fileResource struct {
 
 func (r *fileResource) newKey() *fileResourceKey {
 	return &fileResourceKey{
```

Comment:
I can fix this later, but if you don't mind: I think returning a non-pointer fileResourceKey is better here.

Reply:
Done, though I didn't update the array in the struct because of the

resourceKeys[i] != nil

check